app/selectors.js

```diff
@@ -278,6 +291,109 @@ export const viewedTransaction = createSelector(
   (transactions, txHash) => find({ txHash }, transactions.All)
 );
 
+export const decodedTransactions = get(["grpc", "decodedTransactions"]);
+
+const ticketNormalizer = createSelector(
+  [decodedTransactions, network],
+  (decodedTransactions, network) => {
+    return ticket => {
+      const hasSpender = ticket.spender && ticket.spender.getHash();
+      const isVote = ticket.status == "voted";
+      const ticketTx = ticket.ticket;
+      const spenderTx = hasSpender ? ticket.spender : null;
+      const hash = reverseHash(Buffer.from(ticketTx.getHash()).toString("hex"));
+      const spenderHash = hasSpender ? reverseHash(Buffer.from(spenderTx.getHash()).toString("hex")) : null;
+      const decodedTicketTx = decodedTransactions[hash] || null;
+      const decodedSpenderTx = hasSpender ? (decodedTransactions[spenderHash] || null) : null;
+
+      // effective ticket price is the output 0 for the ticket transaction
+      // (stakesubmission script class)
+      const ticketPrice = ticketTx.getCreditsList()[0].getAmount();
+
+      // ticket tx fee is the fee for the transaction where the ticket was bought
+      const ticketTxFee = ticketTx.getFee();
+
+      // missing: split tx fee
```

Comment:
I'm not entirely sure if I should include the fee for the transaction before the ticket or not.

Reply:
don't worry about the split fee, may be going away entirely soon.